Packs/Jira/Integrations/JiraV2/JiraV2.py

```
@@ -741,6 +743,64 @@ def edit_issue_command(issue_id, mirroring=False, headers=None, status=None, tra
     return get_issue(issue_id, headers, is_update=True)
 
 
+def append_to_field_command(issue_id, fieldJson, headers=None, **kwargs):
+    issue = jira_req('GET', f'rest/api/latest/issue/{issue_id}', resp_type='json')
+
+    for field in fieldJson:
+        field_type = __get_field_type(field)
+        if not field_type:
+            return_error(f"field {field} could not be updated.")
```

Please change to `raise` instead of `return_error`.